.github/workflows/main.yml

```diff
@@ -0,0 +1,34 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+      - name: Install pip
+        run: python -m pip install --upgrade pip
+      - name: Install cython wheel
+        run: pip install cython wheel
+      - name: Install vcf2fhir
+        run: pip install vcf2fhir
```

Comment:
Was there a reason to install our vcf2fhir package from pip ? We would like to run tests on the checked out PR or merged changes and not again the published vcf2fhir package. Can you try raising a PR( containing a single failing test) against your own fork and validate it fails the CI or any other way to make sure this is working.